roles/prosody_setup/tasks/main.yml

```
@@ -29,3 +29,31 @@
     owner: "{{ webapp_user }}"
     group: "{{ webapp_user }}"
     recurse: true
+
+- name: Ensure the presense of nfs-common
+  tags: setup
+  ansible.builtin.apt:
+    name: nfs-common
+    state: present
+
+- name: Ensure the presence of cdh qa mount
+  tags: setup
+  ansible.posix.mount:
+    src: "{{ nfs_server_staging }}:/var/nfs/cdh"
+    path: "/mnt/nfs/cdh"
+    state: mounted
+    fstype: nfs
+    opts: rw,sync,hard
+  when:
+   - "'test' in inventory_hostname"
```

We usually test based on the `qa` variable, which is set in qa vars for all hosts in the qa host group.  (Although we could revisit that; seems like it would be cleaner to have a variable that was set to something like staging/production)

```suggestion
   - qa is defined
```

---

alternately: if we go with my proposal of defining one `nfs_server` variable with different values for qa and prod, the conditional goes away here and we just mount the appropriate server for the environment